### PR TITLE
[Blog]: blog post

### PR DESCRIPTION
**Pull request by chandrashritii**
**Description**

Blog post

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: assets/images/posts/2020-08-06-how-i-schooled-my-seniors/feature-image.svg [hunk 1]
```diff
@@ -0,0 +1,565 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```
Comment (leecalcote) [inline assets/images/posts/2020-08-06-how-i-schooled-my-seniors/feature-image.svg:2]: Unfortunately, the svg is missing the Poppins font used for the tagline. Perhaps, we'll need to use a PNG in this case.

File: collections/_blogs/2020-08-04-how-i-schooled-my-seniors.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+I pitched my idea to [Lee Calcote](https://slack-redir.net/link?url=https%3A%2F%2Ftwitter.com%2Flcalcote&v=3) and in his usual supportive form, he was as excited as I was about the session. A brainstorm session led to creating an entire presentation with suggestions from Lee. The Layer5 community offers a number of resources to contributors, including these two decks ([Meshery - The Service Mesh Management Plane](https://slack-redir.net/link?url=https%3A%2F%2Fdocs.google.com%2Fpresentation%2Fd%2F14kxjwYSJ_FyE3K_6CDEd6oq2kqwn0OSE8RDJ4H-KlKU%2Fedit%3Fusp%3Dsharing&v=3) and [Layer5 - The Service Mesh Community](https://slack-redir.net/link?url=https%3A%2F%2Fdocs.google.com%2Fpresentation%2Fd%2F1tLcMcyQ-Z6YYISYyR0Eyb30dojQiKHIjs7Rt5m8kcag%2Fedit%3Fusp%3Dsharing&v=3)) that I drew from as a basis for a personalized presentation and demonstration to my new team.
```
Comment (NupurThakur27) [inline collections/_blogs/2020-08-04-how-i-schooled-my-seniors.md:15]: Why there is a Slack redirection link to Twitter?

File: collections/_blogs/2020-08-04-how-i-schooled-my-seniors.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+This whole experience has been an amazing journey for me. From being a complete noob to utilizing my expertise in the service mesh domain to landing an actual job, the experience has been a maze of ups and downs. But, I feel like I have finally arrived and arrived with a pot of gold. I’m eternally grateful to Lee Calcote for motivating me and providing me with the appropriate resources. Entering and exploring an open source community can be a daunting task and the [resources provided by Layer5](https://slack-redir.net/link?url=https%3A%2F%2Fdrive.google.com%2Fdrive%2Ffolders%2F1Wg_LPY3ImAexejNFpHWMCBr26UiMUK68%3Fusp%3Dsharing&v=3) helped them in overcoming that challenge and enabled them to learn more about the widespread work of [Meshery](https://slack-redir.net/link?url=https%3A%2F%2Fmeshery.io%2F&v=3).
```
Comment (NupurThakur27) [inline collections/_blogs/2020-08-04-how-i-schooled-my-seniors.md:22]: Link should be directly to Google Drive not via Slack
@chandrashritii Am I right?

Comment (leecalcote) [reply collections/_blogs/2020-08-04-how-i-schooled-my-seniors.md:22]: Right, no, it shouldn't.

Comment (chandrashritii) [reply collections/_blogs/2020-08-04-how-i-schooled-my-seniors.md:22]: Thanks for pointing it out @NupurThakur27. I'll correct it

**Comment by leecalcote** [comment]
The feature image on this post is fitting. 

Badass. 

Just like @NupurThakur27.

**Comment by NupurThakur27** [comment]
> The feature image on this post is fitting.
> 
> Badass.
> 
> Just like @NupurThakur27.

Honestly, I was wondering who worked on that image and wanted to thank the person, it's such an amazing illustration.  🎉

**Comment by Nikhil-Ladha** [comment]
I also want one! Infact a better and interesting one @chandrashritii for my post 😆⚔️.

**Comment by chandrashritii** [comment]
> 
> 
> > The feature image on this post is fitting.
> > Badass.
> > Just like @NupurThakur27.
> 
> Honestly, I was wondering who worked on that image and wanted to thank the person, it's such an amazing illustration. 🎉

I made it. Really happy you like it @NupurThakur27 😎

**Pull request merged**